vanilla-framework/vanilla-dashboard-theme
# Table responsive cards

## Done

- Created new table card style for mobile
- Styled contextual menus to follow the same style as per MAAS

## QA

- Pull down the branch and do `./run` in the terminal
- Check the responsive tables HTML and SCSS
- Check against designs https://github.com/ubuntudesign/vanilla-dashboard-theme-design/blob/master/Table%20responsive/table-responsive.png

## Details

Fixes #25
